homework_2/fastq_filtrator.py

```diff
@@ -0,0 +1,69 @@
+
+def gc_content(read, threshold):
+    read = read.upper()
+    gc_number = 0
+    gc_number += read.count('G')
+    gc_number += read.count('C')
```

Comment:
`count` внутри себя полностью проходит по строке, получается, что мы это делаем два раза, хотя это возможно сделать за один. Но это не критично, зато хорошо читается
```suggestion
    gc_number = read.count('G') + read.count('C')
```